website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
```

Comment:
Is it a "_a_ database engine" or "_the_ database engine"? Half of our docs say "a" and another half say "the". I think we should pick one or another.